docs/operator-guide/cd/manual-approval.md

```diff
@@ -0,0 +1,125 @@
+# Manual Approval in Deploy Pipelines
+
+KubeRocketCI features manual approval task in deploy pipelines, which stops the pipeline execution until the user makes a decision or pipeline is timed out. This feature allows to integrate a quality control checkpoint in your pipelines, reducing risks when deploying applications to production environments.
```

Comment:
let's rephrase this, there's no need to focus on the fact that this is a deployment pipeline, let's emphasize that this is additional verification when promoting an application from lower application levels to production.